metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py

```diff
@@ -1131,7 +1132,19 @@ def from_api(  # noqa: C901
                 logger.warning(
                     f"Failed to extract explore {explore_name} from model {model}: {e}"
                 )
-
+        except DeserializeError as e:
+            logger.warning(
+                f"Failed to extract explore {explore_name} from model {model}, Error: {e}"
+            )
+            reporter.warning(
+                title=f"Failed to extract explore: {explore_name}",
```

Comment:
the title should be a bit more descriptive on what exactly went wrong, and shouldn't have any templating

```suggestion
                title="Failed to fetch explore from the Looker API",
```